tabbycat/results/views.py

```diff
@@ -620,6 +614,11 @@ def get_context_data(self, **kwargs):
 
 class PrivateUrlBallotScoresheetView(RoundMixin, SingleObjectByRandomisedUrlMixin, BasePublicBallotScoresheetsView):
 
+    slug_url_kwarg = 'url_key'
+
+    def get_slug_field(self):
+        return 'debateadjudicator__adjudicator__url_key'
```

Comment:
I think it works if you just set the `slug_field` property of the class, right? Since this doesn't need to be generated dynamically?

Reply:
You're right